Makefile

```diff
@@ -2,7 +2,8 @@ bootstrap:
 	gem install bundler -v 2.0.2
 	bundle install
 	# pod install
-	bundle exec pod keys set BaseAPIURL "https://polydice.com/iCook-tvOS/demo/"
+	bundle exec pod keys set BaseAPIURL "https://bcyl.in/Try-tvOS/demo/"
+	# "https://polydice.com/iCook-tvOS/demo/": An SSL error has occurred and a secure connection to the server cannot be made.
```

Comment:
Nice catch. How about using jsdelivr for this?
i.e. https://cdn.jsdelivr.net/gh/polydice/iCook-tvOS@gh-pages/demo/

Reply:
Sounds good 👍 I’ll update it.